### PR TITLE
ST-712 reworked

### PR DESCRIPTION
**Pull request by tamir-b**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-712

### Change description ###
Reworked ST-712 to not show if Representative is not present

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by osheaf01 — COMMENTED** [review]
Just one comment - why is the field being commented out in one of the classes?

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecision.java [hunk 1]
```diff
@@ -40,8 +36,8 @@ public class CaseWorkerIssueDecision implements CCDConfig<CaseData, State, UserR
     private static final CcdPageConfiguration issueDecisionUploadNotice = new IssueDecisionUploadNotice();
     private static final CcdPageConfiguration issueDecisionSelectRecipients = new IssueDecisionSelectRecipients();
 
-    @Autowired
-    private DecisionIssuedNotification decisionIssuedNotification;
+    /*@Autowired
+    private DecisionIssuedNotification decisionIssuedNotification;*/
```
[the remark this answers is not in the document]
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecision.java:40]: Please dont comment this code. we are still in progress with testing notifications in AAT env. Please uncomment.

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecision.java [hunk 2]
```diff
@@ -87,29 +83,10 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
...
-        }
-
-        if (caseIssueDecision.getRecipients().contains(ContactPartiesCIC.RESPONDANTTOCONTACT)) {
-            decisionIssuedNotification.sendToRespondent(data, caseNumber);
-        }
-
-        if (caseIssueDecision.getRecipients().contains(ContactPartiesCIC.REPRESENTATIVETOCONTACT)) {
-            decisionIssuedNotification.sendToRepresentative(data, caseNumber);
-        }
-    }
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecision.java:114]: Please re-add the code, this is related to notifications.

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecision.java [hunk 3]
```diff
@@ -87,29 +83,10 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
     public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
                                                CaseDetails<CaseData, State> beforeDetails) {
 
-        CaseIssueDecision caseIssueDecision = details.getData().getCaseIssueDecision();
-        var message = MessageUtil.generateIssueDecisionMessage(caseIssueDecision);
-
-        sendIssueDecisionNotification(details.getData().getHyphenatedCaseRef(), details.getData());
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecision.java:93]: Please add line 93 - Part of notification.

**Review by ElifDenizEsea — APPROVED** [review]

**Pull request merged**